de.dlr.sc.overtarget.language.ui/src/de/dlr/sc/overtarget/language/ui/editor/UpdateUnitManagerAction.xtend

```diff
@@ -34,20 +34,22 @@ class UpdateUnitManagerAction extends TextEditorAction {
 			val input = editor.editorInput
 			if (input instanceof IFileEditorInput) {
 
-			// get current document from editor which represents the tmodel
-			val overtargetEditor = new OvertargetXtextEditor
-			val document = overtargetEditor.getXtextDocument
-			if (document !== null) {
+				// get current document from editor which represents the tmodel
+				val overtargetEditor = new OvertargetXtextEditor
+				val document = overtargetEditor.getXtextDocument
+				if (document !== null) {
 					val target = document.contents.get(0) as TargetFile
 					val repositoryLocations = target.repositoryLocations
 					for (reposLoc : repositoryLocations) {
 						val unitManager = UnitManager.instance
-						if (!unitManager.checkIfUnitsLoaded(reposLoc.name)) {
-							unitManager.loadUnits(reposLoc)
+						if (unitManager.checkUriIsValid(reposLoc)) {
```

Comment:
Please check indentation of curly brackets. Looks like this if statement is opening a scope with a curly bracket but it doesn'T have the closing one on the expected level.